### ISSUE TITLE
Upgrade our dropdown component

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#116]
## Description
User tests showed some users struggle to recognize the component as a dropdown. 

## Possible changes
- A placeholder text ("select country")
- A larger down arrow or other visual change do make it look more unique

## Tasks
- [ ] Decide new design specification
- [ ] Update Figma

**Comment by mimarz** [comment]
Is this still relevant for our new Combobox @Febakke ?

**Comment by Febakke** [comment]
No, I hope its pretty clear with the chevron. We can re-open this if we get any complaints.

**Comment by mimarz** [comment]
hmm, I cant close this as its in the old repo. Mind closing it?

**Comment by Febakke** [comment]
We have followed standard combobox/Select designs in our new component. Closing this for now.
